src/utilities.cpp

```diff
@@ -59,7 +59,7 @@ void pasteFilesFromClipboard(FmPath* destPath, QWidget* parent) {
   if(data->hasFormat("x-special/gnome-copied-files")) {
     // Gnome, LXDE, and XFCE
     QByteArray gnomeData = data->data("x-special/gnome-copied-files");
-    char* pdata = gnomeData.data();
+    const char* pdata = gnomeData.constData();
```

Comment:
@tsujan I was able to build it with clang. Did you use gcc ?
The prototype is `char *strchr(const char *s, int c);` anyway. Will check it later.